### DISCUSSION TITLE
How can I log in to Coder.com?

### DISCUSSION THREAD

**Discussion by lg-nineoneninethreenine** [coder/code-server#3915] [Q&A] [answered]
I signed up for Coder as trial. I tried looking for the login button but couldn't find it anywhere. How can I log in to Coder?

**Comment by jsjoeio** [comment]
Hmm...that's strange. Sorry about that! Let's see if we can help. Mind sending me an email joe@coder.com and letting me know which email you signed up with and what day?

**Reply by lg-nineoneninethreenine** [reply]
Well, I signed up with a Vivaldi Webmail Account but deleted it accidentally. I will sign up again and let you know. (Vivaldi Webmail account only)

**Reply by jsjoeio** [reply]
Sounds good!

**Reply by lg-nineoneninethreenine** [reply]
> Sounds good! 

Mailed you

**Reply by jsjoeio** [reply]
@bpmct should be in touch soon to help you!

**Reply by lg-nineoneninethreenine** [reply]
No mail yet. Did @bpmct get a prompt that he was mentioned in this discussion? If not,

/ping @bpmct

**Comment by bpmct** [comment, marked as answer]
Whoops! Not sure how I missed the initial notification. I found your license and just sent an email with instructions :)

**Reply by lg-nineoneninethreenine** [reply]
Thanks @bpmct I will set it up soon on something like AWS or Oracle.
